DrinkManagerConsole/ReviewUi.cs

```diff
@@ -0,0 +1,43 @@
+﻿using BLL;
+using System;
+
+namespace DrinkManagerConsole
+{
+    public static class ReviewUi
+    {
+        public static void EditReview(Drink drink)
```

Comment:
I think it'd be better to call this method GetDataFromUserForNewReview, because now it's the same as EditReview from ReviewService.